Example/Tests/HaventecAuthenticateTests.swift

```diff
@@ -80,6 +80,29 @@ class HaventecAuthenticateTest: XCTestCase {
         XCTAssertThrowsError(try HaventecAuthenticate.getUserUuid())
     }
 
+    func testEdgeCases() throws {
```

Comment:
Add edge test for:
- Hard coded hashPin that's of proper length but not valid value to any device (e.g. 'AAAA....')

Reply:
Added extra base64 asserts